api/src/test/java/org/openmrs/util/databasechange/Database1_9_7UpgradeIT.java

```diff
@@ -161,13 +160,12 @@ public void shouldUpgradeFromClean1_9To1_10() throws IOException, SQLException {
 	public void shouldFailMigratingDrugOrdersIfUnitsToConceptsMappingsIsNotSet() throws IOException, SQLException {
 		upgradeTestUtil.executeDataset(STANDARD_TEST_1_9_7_DATASET);
 		createOrderEntryUpgradeFileWithTestData("");
-		expectedException.expect(IOException.class);
 		String errorMsgSubString1 = "liquibase.exception.MigrationFailedException: Migration failed for change set liquibase-update-to-latest.xml::201401101647-TRUNK-4187::wyclif";
-		expectedException.expectMessage(errorMsgSubString1);
 		String errorMsgSubString2 = Context.getMessageSourceService().getMessage("upgrade.settings.file.not.have.mapping",
 		    new Object[] { "mg" }, null);
-		expectedException.expectMessage(errorMsgSubString2);
-		upgradeTestUtil.upgrade();
+		IOException exception = assertThrows(IOException.class, () -> upgradeTestUtil.upgrade());
+		assertThat(exception.getMessage(), is(errorMsgSubString1));
+		assertThat(exception.getMessage(), is(errorMsgSubString2));
```

Comment:
How can the message be equal to 2 different error messages? I believe this should fail when running the test. Doesnt it?
Remember that expectMessage is checking whether a substring is contained in the message. So instead of is which checks equality it should be containsString I assume